docs/data/stashcache/run-stashcache-container.md

```diff
@@ -44,24 +129,44 @@ XC_RESOURCENAME=<YOUR_RESOURCE_NAME>
 CACHE_FQDN=<FQDN>
 ```
 
+### Ensure the xrootd service has a certificate
+
+The service will need a certificate for contacting central OSDF services
+and for authenticating to origins.
+
+The certificate and key must be inside the container at `/etc/grid-security/xrd/xrd{cert,key}.pem`,
+owned by the `xrootd` user and group (UID/GID 10940).
+
+The easiest solution for this is to use your host certificate and key.
+Volume-mount the certificate to `/etc/grid-security/hostcert.pem` and the key to `/etc/grid-security/hostkey.pem`.
+The files will be copied with the right permissions on container startup.
+
+!!! note
+    You must restart the container whenever you renew your certificate
+    in order for the services to pick up the new certificate.
+    If you automate certificate renewal, you should automate restarts as well.
+    For example, if you are using Certbot for Let's Encrypt, you should write a "deploy hook" as documented
+    [on the Certbot site](https://certbot.eff.org/docs/using.html#renewing-certificates).
+
+
 ### Optional configuration ###
 
 Further behavior of the cache can be configured by setting the following in the environment variable file:
 
-- `XC_SPACE_HIGH_WM`, `XC_SPACE_LOW_WM`: High-water and low-water marks for disk usage;
-      when usage goes above the high-water mark, the cache will delete files until it hits the low-water mark.
-- `XC_PORT`: TCP port that XCache listens on
+- `XC_SPACE_HIGH_WM`, `XC_SPACE_LOW_WM`: High-water and low-water marks for disk usage,
+  as numbers between 0.00 (0%) and 1.00 (100%);
+  when usage goes above the high-water mark, the cache will delete files until it hits the low-water mark.
+- `XC_PORT`: TCP port that XCache listens on.
```

Comment:
We list three ports above so I think this needs clarification on which port this controls

Reply:
... a bit of grepping shows that XC_PORT is an atlas-xcache and cms-xcache thing, so I will drop it entirely.